compiler/rustc_codegen_llvm/src/debuginfo/metadata.rs

```diff
@@ -473,6 +470,7 @@ pub(crate) fn spanned_type_di_node<'ll, 'tcx>(
             AdtKind::Enum => enums::build_enum_type_di_node(cx, unique_type_id, span),
         },
         ty::Tuple(_) => build_tuple_type_di_node(cx, unique_type_id),
+        ty::Field(..) => build_field_type_di_node(cx, unique_type_id),
```

Comment:
Since this is effectively a never type, why not group it with `Never` here?

(I know nothing about debug info.^^)